server/starters/web-starter/src/main/java/com/oceanbase/odc/service/info/WebInfoAdapter.java

```diff
@@ -66,7 +66,7 @@ public String getLoginUrl(HttpServletRequest request) {
             return alipayOpenApiProperties.getObOfficialLoginUrl();
         } else if (authType.contains("local")) {
             SSOIntegrationConfig sSoClientRegistration = integrationService.getSSoIntegrationConfig();
-            if (sSoClientRegistration == null || !sSoClientRegistration.isOauth2OrOidc()) {
+            if (sSoClientRegistration == null) {
```

Comment:
What's this change for?

Reply:
Adapte for saml,  the type should not be distinguished  here,  but inside method.